vendor/github.com/prometheus/alertmanager/config/notifiers.go

```diff
@@ -694,6 +689,7 @@ type PushoverConfig struct {
 	Priority    string   `yaml:"priority,omitempty" json:"priority,omitempty"`
 	Retry       duration `yaml:"retry,omitempty" json:"retry,omitempty"`
 	Expire      duration `yaml:"expire,omitempty" json:"expire,omitempty"`
+	TTL         duration `yaml:"ttl,omitempty" json:"ttl,omitempty"`
```

Comment:
We need to let the frontend folks know that this option is now support in Mimir so that Grafana can be adjusted. cc: @gillesdemey